### DISCUSSION TITLE
Welcome to netty-incubator-codec-http3 Discussions!

### DISCUSSION THREAD

**Discussion by NiteshKant** [netty/netty-incubator-codec-http3#172] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hernansaab** [comment]
I am looking for an example on how to use the QPACK codec without HTTP3 context. I basically would like to apply QPACK directly to the UDP datagrams. Any help would be appreciated.

**Reply by normanmaurer** [reply]
This is not possible atm
